requirements.txt

```diff
@@ -17,5 +17,5 @@ pyproj==1.9.3
 prettytable==0.7.2
 # When updating nupic.bindings, also update any shared dependencies to keep
 # versions in sync.
-nupic.bindings==0.4.6
+nupic.bindings>=0.4.6
```

Comment:
I think that the idea, communicated to me by @scottpurdy IIRC, was that the new scheme would enable nupic.bindings API changes to get ahead of nupic temporarily. The `>=` might create conflicts by picking up a newer, potentially incompatible, version of nupic.bindings when installing a particular version of of nupic.